code/game/objects/items/clothing/clothing.dm

```diff
@@ -403,7 +399,6 @@ BLIND	 // can't see anything
 		I.forceMove(src)
 		holding = I
 		user.visible_message("<span class='notice'>\The [user] shoves \a [I] into \the [src].</span>")
-		verbs |= /obj/item/clothing/shoes/proc/draw_knife
```

Comment:
This and other changes in this file isn't related to to_chat

Reply:
this was unintended